### PR TITLE
Add partial functionality to filterByNameOrIng method

### PR DESCRIPTION
**Pull request by aliroemhildt**
#### What's this PR do?
filterByNameOrIng can now filter by ingredient, doesn't filter by name yet

#### Where should the reviewer start?
line 9 of RecipeRepository.js

#### How should this be manually tested?
have only tested with console.logs so far

#### Any background context you want to provide?
n/a

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
n/a

#### Questions:
n/a

### CODE REVIEW

**Review by kammon10 — COMMENTED** [review]
Impressive work!!

File: src/classes/RecipeRepository.js [hunk 1]
```diff
@@ -1,34 +1,90 @@
...
-  //function that searches ing array and returns array of ids
-  let getIDfromName = ingArray.filter((element) => {
-      return element.name.includes(nameOrIng)
-    });
-  //call function from above and store arry in var idList
-  let idList = getIDfromName().map((element) => {
-    return element.id
-  })
+  filterByNameOrIng(nameOrIng) {
+    // 1. create empty array that will hold recipes we want to display (from search)
```
Comment (anewb87) [inline src/classes/RecipeRepository.js:10]: Really happy with how much we're psuedocoding to be able to SEE our thought processes.

**Review by cagallo — APPROVED** [review]
Looks great! We are so close!

**Review by anewb87 — COMMENTED** [review]
Holy doozy of a method! This is a beast. Feeling pretty positive though.

**Pull request merged**